### PR TITLE
#489 make taupage config option naming consistent 

### PR DESCRIPTION
**Pull request by smirnov**
for enabling introspection, remove faulty support

### CODE REVIEW

File: runtime/etc/init/planb-tokeninfo.conf [hunk 1]
```diff
@@ -14,7 +14,7 @@ script
         export OPENID_PROVIDER_CONFIGURATION_URL="$config_openid_provider_configuration_url"
         export REVOCATION_PROVIDER_URL="$config_revocation_provider_url"
         export BUSINESS_PARTNERS="$config_tokeninfo_business_partners"
-        if [ "$config_tokeninfo_introspection" = "true" ] || [ "$config_tokeninfo_introspection" = "True" ]; then
+        if [ "$config_planb_local_tokeninfo_introspection" = "true" ] || [ "$config_planb_local_tokeninfo_introspection" = "True" ]; then
```
[the remark this answers is not in the document]
Comment (jbspeakr) [reply runtime/etc/init/planb-tokeninfo.conf]: iMO it should be `local_planb_tokeninfo: true` and `local_planb_tokeninfo_introspection: true`, yes

[the remark this answers is not in the document]
Comment (smirnov) [reply runtime/etc/init/planb-tokeninfo.conf]: Addressed.

**Comment by smirnov** [comment]
:+1:

**Comment by jbspeakr** [comment]
👍

**Comment by hjacobs** [comment]
Is this option documented on http://stups.readthedocs.io ?

**Pull request merged**